### ISSUE TITLE
i need my toast to not auto dismiss when I hover over it.

### ISSUE THREAD

**Issue by geetee24** [PointInside/ng2-toastr#204]
Please

Is there a way this can be done?

Thanks.

**Comment by gregor-srdic** [comment]
I believe I have a similar question. Is it possible to keep the toast opened also if app is redirected to another page via the angular router navigate method?

**Comment by SergBozhko** [comment]
I need it too

**Comment by Ecksters** [comment]
Yeah, would be nice to have an option for "retainOnHover" or something along those lines. Perhaps 2 different options, 1 that makes it turn into a manual dismiss on hover, and one that just makes it only dismiss on mouseout.
